sdk/videoanalyzer/azure-media-videoanalyzer-edge/README.md

```diff
@@ -0,0 +1,202 @@
+# Azure Video Analyzer Edge client library for Java
+
+Azure Video Analyzer on IoT Edge provides a platform to build intelligent video applications that span the edge and the cloud. The platform offers the capability to capture, record, and analyze live video along with publishing the results, video and video analytics, to Azure services in the cloud or the edge. It is designed to be an extensible platform, enabling you to connect different video analysis edge modules (such as Cognitive services containers, custom edge modules built by you with open-source machine learning models or custom models trained with your own data) to it and use them to analyze live video without worrying about the complexity of building and running a live video pipeline.
+
+Use the client library for Video Analyzer on IoT Edge to:
+
+- Simplify interactions with the [Microsoft Azure IoT SDKs](https://github.com/azure/azure-iot-sdks)
+- Programmatically construct pipeline topologies and live pipelines
+
+[Product documentation][doc_product] | [Direct methods][doc_direct_methods] | [Source code][source]
+
+## Getting started
+
+### Install the package
```

Comment:
We title this as `Include the package` in Java README and also include the maven dependency for this package.

For example - https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/appconfiguration/azure-data-appconfiguration#include-the-package